tests/portage_utils.py/Makefile

```diff
@@ -103,25 +103,28 @@ open_unittest7: open_unittest7.gz test
 	zcmp $+
 open_unittest7.gz: test
 	cat $< \
-	| python -c "exec('from portage_utils import open\nf=open(\"| gzip > $@\", \"w\")\nfor line in open(\"-\"): f.write(line)')"
+	| python3 -c "exec('from portage_utils import open\nf=open(\"| gzip > $@\", \"w\")\nfor line in open(\"-\"): f.write(line)')"
 
 
 # Test writing to no file which should produce an error message.
 open_unittest8: test
 	cat $< \
-	| python -c "exec('from portage_utils import open\nf=open(\"\", \"w\")\nfor line in open(\"-\"): f.write(line)')" 2>&1 \
+	| python3 -c "exec('from portage_utils import open\nf=open(\"\", \"w\")\nfor line in open(\"-\"): f.write(line)')" 2>&1 \
 	| grep "Fatal error: You must provide a filename" --quiet
 
 
 .PHONY: open_unittest9
 open_unittest9:  open_unittest9a open_unittest9b
 
+big.gz:
+	seq 1 1000000 | gzip > $@
+
 # Testing partially reading a gzip file and not getting a Broken pipe message.
-open_unittest9a:  %:  ${PORTAGE}/test-suite/systems/toy-regress-en2fr/models/tm/cpt.merged.hmm3-kn3-zn.tm-train.en2fr.gz
-	python -c "exec('from portage_utils import open\nfor line in open(\"$<\", \"r\", False):\n  if True: break\n')" 2>&1 | egrep '(zcat|gzip): stdout: Broken pipe'
+open_unittest9a:  %:  big.gz
+	-python3 -c "exec('from portage_utils import open\nfor line in open(\"$<\", \"r\", False):\n  if True: break\n')" 2>&1 | egrep '(zcat|gzip): stdout: Broken pipe'
 
-open_unittest9b:  %:  ${PORTAGE}/test-suite/systems/toy-regress-en2fr/models/tm/cpt.merged.hmm3-kn3-zn.tm-train.en2fr.gz
```

Comment:
I really like that we get ride of `${PORTAGE}` which was project specific.

Reply:
Yeah, I really wanted that to go away! :)